src/quacc/runners/ase.py

```diff
@@ -362,3 +369,171 @@ def _set_sella_kwargs(self, optimizer_kwargs: dict[str, Any]) -> None:
+        dyn.run(fmax=fmax, steps=max_steps, **run_kwargs)
+
+    # Store the trajectory atoms
+    dyn.traj_atoms = read(traj_file, index=":")
+
+    # Perform cleanup operations
+    for ii, image in enumerate(images):
+        calc_cleanup(image, dir_lists[ii][0], dir_lists[ii][1])
+
+    return dyn
+
+
+def _geodesic_interpolate_wrapper(
+    reactant: Atoms,
+    product: Atoms,
+    nimages: int = 20,
+    perform_sweep: bool | None = None,
+    convergence_tolerance: float = 2e-3,
+    max_iterations: int = 15,
+    max_micro_iterations: int = 20,
+    morse_scaling: float = 1.7,
+    geometry_friction: float = 1e-2,
+    distance_cutoff: float = 3.0,
+) -> tuple[list[str], list[list[float]]]:
+    """
+    Interpolates between two geometries and optimizes the path with the geodesic method.
+
+    Parameters
+    ----------
+    reactant_product_atoms : List[Atoms]
+        List of ASE Atoms objects containing initial and final geometries.
+    nimages : int, optional
+        Number of images for interpolation. Default is 20.
+    perform_sweep : Optional[bool], optional
+        Whether to sweep across the path optimizing one image at a time.
+        Default is to perform sweeping updates if there are more than 35 atoms.
+    convergence_tolerance : float, optional
+        Convergence tolerance. Default is 2e-3.
+    max_iterations : int, optional
+        Maximum number of minimization iterations. Default is 15.
+    max_micro_iterations : int, optional
+        Maximum number of micro iterations for the sweeping algorithm. Default is 20.
+    morse_scaling : float, optional
+        Exponential parameter for the Morse potential. Default is 1.7.
+    geometry_friction : float, optional
+        Size of friction term used to prevent very large changes in geometry. Default is 1e-2.
+    distance_cutoff : float, optional
+        Cut-off value for the distance between a pair of atoms to be included in the coordinate system. Default is 3.0.
+
+    Returns:
+    --------
+    Tuple[List[str], List[List[float]]]
+        A tuple containing the list of symbols and the smoothed path.
+    """
+    # Read the initial geometries.
+    chemical_symbols = reactant.get_chemical_symbols()
+
+    # First redistribute number of images. Perform interpolation if too few and subsampling if too many images are given
+    raw_interpolated_positions = redistribute(
+        chemical_symbols,
+        [reactant.positions, product.positions],
+        nimages,
+        tol=convergence_tolerance * 5,
+    )
+
+    # Perform smoothing by minimizing distance in Cartesian coordinates with redundant internal metric
+    # to find the appropriate geodesic curve on the hyperspace.
+    geodesic_smoother = Geodesic(
+        chemical_symbols,
+        raw_interpolated_positions,
+        morse_scaling,
+        threshold=distance_cutoff,
+        friction=geometry_friction,
+    )
+    if perform_sweep is None:
+        perform_sweep = len(chemical_symbols) > 35
```

Comment:
Where does this 35 atom cutoff come from?

Reply:
From geodesic's own code.